syntax = "proto3";

message requestQuery{
  string id = 1;
  string db_name = 2;
  string op = 3;
}

message results {
  bytes msg = 1;
}

service RequestTransmission {
  rpc RequestParsing (requestQuery) returns (results);
  rpc RequestDecrypt (results) returns(results);
}